tools/ci_connector_ops/setup.py

```diff
@@ -21,7 +21,8 @@
         "console_scripts": [
```

Comment:
Would you mind bumping the package version? It's not released anywhere so versionning it a good practice only at the moment.